client/state/happychat/actions.js

```diff
@@ -79,35 +81,62 @@ export const requestTranscript = () => ( dispatch, getState ) => {
 	);
 };
 
+const getSettings = new Promise( resolve => {
+	if ( userSettings.initialized ) {
+		return resolve( userSettings );
+	}
+	userSettings.once( 'change', () => {
+		resolve( userSettings );
+	} );
+} );
+
+const DEFAULT_LOCALE = 'en';
+
+const getLocale = getSettings.then( settings => {
+	const olarkState = olarkStore.get();
+	const olarkLocale = olarkState.locale;
+	const accountLocale = settings.getSetting( 'language' );
+
+	if ( accountLocale === DEFAULT_LOCALE ) {
+		debug( 'using olark chat locale' );
+		return olarkLocale;
+	}
+
+	return accountLocale;
+} );
```

Comment:
> I'm also curious about the fallback to the olark locale. Is there a reason we couldn't just use en if that's the account locale?

I'm happy with removing the olark specific setting. I was under the impression that some users may have this setting set.

> Would the happychat service handle that fallback, if we pass ja as the locale?

Yes. If we send a locale that is not explicitly supported by happychat, happychat will open a chat with the happychat system default locale (in our case `en`).

It the user is set to a supported locale (e.g. `pt-br`) and there is no available capacity on happychat for that locale, happychat will not be enabled for their current session.